cmd2/cmd2.py

```diff
@@ -177,14 +177,38 @@ def cmd_wrapper(cmd2_app, statement: Union[Statement, str]):
         return arg_decorator
 
 
-def with_argparser_and_unknown_args(argparser: argparse.ArgumentParser, *,
+# noinspection PyProtectedMember
+def set_parser_prog(parser: argparse.ArgumentParser, prog: str):
```

Comment:
Would this utility function be more appropriate in **argparse_custom.py** or here?

Actually, it begs the question would all of the decorators and associated utility functions belong better in a separate file?